artem/HTML_CSS/index.html

```diff
@@ -0,0 +1,661 @@
+                                        <img src="./img/icons/star.svg" alt="#">
+                                        <img src="./img/icons/star.svg" alt="#">
+                                        <img src="./img/icons/star.svg" alt="#">
+                                        <img src="./img/icons/star.svg" alt="#">
+                                    </div>
+                                </div>
+                            </div>
+                        </div>  
+                    </a>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section class="eco">
+        <div class="container">
+            <div class="eco_content">
+                <div class="eco_content_left">
+                </div>
+                <div class="eco_content_right_wrapper">
+                    <div class="eco_content_right_content">
+                        <div class="eco_content_right_text">
+                            <h2>Eco Friendly</h2>
+                            <h1>
+                                From our Farm
+                                <br>
+                                To Your Home.
+                            </h1>
+                        </div>
+                        <div class="eco_content_right_block">
+                            <h3>Choose Your Products</h3>
+                            <p>
+                               In our listing, we have several collections of organic products. 
+                               This is the place where you need to choose the product you want.
+                            </p>
+                        </div>
+                        <div class="eco_content_right_block">
+                            <h3>Farmers Will Produce It</h3>
+                            <p>
+                                The Product that you ordered will be verified that we have or not if have we will start to move on with the next step.
+                            </p>
+                        </div>
+                        <div class="eco_content_right_block">
+                            <h3>We Can Delivery Too</h3>
+                            <p>
+                                If you are not comfortable going to the nearby market place we also will deliver your product to your doorstep.
+                            </p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section class="categ">
+        <div class="categ_fruit width_height">
+            <a href="#" class="categ_button">Fruit Juice</a>
+        </div>
+        <div class="categ_organic width_height">
+            <a href="#" class="categ_button">Organic Food</a>
+        </div>
+        <div class="categ_nuts width_height">
+            <a href="#" class="categ_button">Nuts Cookies</a>
+        </div>
+    </section>
+    <section class="categ1">
```

Comment:
bad parctic is create new block at different @media with delete this section and change styles for section categ at other @media